models/_sector/alm/ethereum/alm_ethereum_base_trades.sql

```diff
@@ -0,0 +1,62 @@
+{{ config(
+    schema = 'alm_ethereum'
+    , alias = 'base_trades'
+    , materialized = 'view'
```

Comment:
If there is `incremental` in the spell, we may use `materialized = 'incremental'`.